src/components/input/textInput/textInput.cy.tsx

```diff
@@ -1,8 +1,8 @@
-import { supportedViewports } from '../../../cypress/support/variables'
+import { supportedViewports } from '../../../../cypress/support/variables'
 import TextInput from './textInput'
 
 describe('TextInput', () => {
-  const textInput = '[data-cy=text-input]'
+  const textField = '[data-cy=text-field]'
```

Comment:
why is this being changed ? I think it was intentional that the data-cy value would match the component

Reply:
This is targeting the MUI's `TextField` to enter text content. So `text-field` is more accurate

Reply:
why are we targeting MUI TextField in our tests ?

Reply:
We want to type something in MUI's TextField in the tests. I guess I could try to find `textarea` instead.

Reply:
no, why cant we just use our component and add text there?

Reply:
I can use `base-input` instead. Tried to add `text-input` as data-cy, but since textInput only has baseInput, data-cy didn't apply properly.